app/helpers/chat_channels_helper.rb

```diff
@@ -0,0 +1,44 @@
+module ChatChannelsHelper
+  def unopened_json_response
```

Comment:
These names always bothered me, because they sound like they should really be serializers. **But** they don't actually return a JSON response, but `ActiveRecord::Relation`s and helpers are definitely not the place to make DB complex  DB queries.

Reply:
I was thinking to move them to the services or model layer.  In coming up PR we will refactor our controller also. Because our controllers are fat.